tools/build/Docker/README.conda.rst

```diff
@@ -9,7 +9,7 @@ Build and run the `freecad-conda` container with the following commands:
 
 .. code-block:: console
 
-    docker build --file tools/build/Dockerfile.Conda --tag freecad-conda
+    docker build --file tools/build/Docker/Dockerfile.Conda --tag freecad-conda .
```

Comment:
Your change to the path is correct but I think you included a `.` by mistake. It's not there in any of the other `docker build` commands in the main docker readme `tools/Build/Docker/Readme.rst`

Reply:
If the dot is correct, then we should add it to all the other docker build commands too, right?

Reply:
Yes, I added the dot tentatively, because the command didn't work and syntax error indicated the PATH was needed:
` docker buildx build [OPTIONS] PATH | URL | - `

I am not sure why it wasn't there in the first place.